images/tools/sonataflow-db-migrator/pom.xml

```diff
@@ -128,4 +169,16 @@
             </properties>
         </profile>
     </profiles>
+
+    <repositories>
+        <repository>
+            <id>central</id>
+            <name>Maven Central</name>
+            <layout>default</layout>
+            <url>https://repo1.maven.org/maven2</url>
```

Comment:
Why do we need this? Ideally, we should use Apache instead.

Reply:
Changed the repo to make use of: https://repo.maven.apache.org/maven2/